admin/admin-domain-service/ballerina/resources/admin-api.yaml

```diff
@@ -2722,11 +2722,11 @@ paths:
 ######################################################
   # The "Organization" resource API
 ######################################################
-  /organization:
+  /organizations:
     get:
       tags:
         - Organization (Collection)
-      summary: Get all Organization
+      summary: Get all API Categories
```

Comment:
API Categories -> Organisations, there are few places changed with new commit as well, shall we fix?